demo/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <title>Substrait viewer demo</title>
+    <link href='https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css' rel='stylesheet'>
```

Comment:
Ideally there should be a way to include this in `style.scss` (it's kind of like static linking) so it will grab it at build time and embed it in the project files.

Reply:
Made the change. Thanks!